phpBB/styles/prosilver/theme/buttons.css

```diff
@@ -152,6 +152,16 @@
 	max-width: 40%;
 }
 
+/* Format buttons
+--------------------------------------------- */
+.format-buttons .button {
+	margin-bottom: 3px;
+}
+
+.format-buttons input, .format-buttons select {
```

Comment:
I think you misunderstood me this should remain where it was but should also have the margin bottom added.